src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/OrderPreferences/GiftOptionsType.php

```diff
@@ -77,7 +77,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('enable_gift_wrapping', SwitchType::class, [
                 'required' => false,
                 'label' => $this->trans('Offer gift wrapping', 'Admin.Shopparameters.Feature'),
-                'help' => $this->trans('Suggest gift-wrapping to customers.', 'Admin.Shopparameters.Help'),
+                'help' => $this->trans('Suggest gift-wrapping to customers. Remember to regenerate e-mails template if you activate or deactivate this option', 'Admin.Shopparameters.Help'),
```

Comment:
```suggestion
                'help' => $this->trans(Remember to regenerate email templates after enabling or disabling this option.', 'Admin.Shopparameters.Help'),
```

I removed the first sentence as it was a repetition of the label. Also, maybe we could specify which templates should be regenerated?

Reply:
As far as I remember you can't chose a specific one, the regeneration process will regenerate all the e-mails (Design > Email theme)
Maybe we could add the path of the BO, like:
`Remember to regenerate email templates after enabling or disabling this option in Design > Email theme`

Reply:
Good idea, then why not add a link? :-)

`Remember to regenerate email templates after enabling or disabling this option in [1]Design > Email theme[\1].`

Reply:
Done 👍🏻